packages/istanbul-lib-source-maps/lib/source-store.js

```diff
@@ -1,78 +1,80 @@
-var util = require('util'),
-    os = require('os'),
-    path = require('path'),
-    mkdirp = require('make-dir'),
-    rimraf = require('rimraf'),
-    fs = require('fs');
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+const mkdirp = require('make-dir');
+const rimraf = require('rimraf');
 
-function SourceStore(/*opts*/) {}
-
-SourceStore.prototype.registerSource = function(/* filePath, sourceText */) {
-    throw new Error('registerSource must be overridden');
-};
-
-SourceStore.prototype.getSource = function(/* filePath */) {
-    throw new Error('getSource must be overridden');
-};
+class SourceStore {
```

Comment:
If this is meant to be an abstract class, we should have some sort of enforcement:

```js
class SourceStore {
  constructor() {
    if (this.constructor === SourceStore) throw new Error('Abstract class SourceStore must be extended');
  }
}
```

Reply:
Not sure what happened to my comment about this.  I've updated this class to be completely empty, created a noop `MemoryStore.dispose()`.  It's not exported and is only used as a shared base class.  I'm not sure it needs to exist but for now I figure keep the prototype chain the same.